CMSgov/mint-app
# [EASI-3901] Add demands recoupments frequency question

# EASI-3901

## Changes and Description

### BE Changes
- Added new field to schema - `paymentDemandRecoupmentFrequency`
- Added `Continually`, `Other`, and `Note` string fields
- Added migration to add column on `payment_demand_recoupment_frequency`
- Added test for new field

### FE Changes
- Added new field `paymentDemandRecoupmentFrequency` to queries
- Implemented `FrequencyForm` for new question
- Added to Readonly and CSV
- Updated unit tests
- Added to `IPC` filter group



## How to test this change

1. Navigate to page 3 of Payment
2. Make selections the frequency question
3. Verify data persists on save
4. Verify render on readonly
5. Verify csv export data
6. Verify that the field is part of the IPC filter view



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
